tools/slither-analysis/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM trailofbits/eth-security-toolbox
```

Comment:
Could you tell me the purpose of this dockerfile? Is it required to be included? is it gonna be utilized in CI?